src/components/App.vue

```diff
@@ -1,7 +1,47 @@
+<!-- Created a basic footer, I had to look into Bootstrap becuase I am more familiar with Foundation for styling  -->
+<!-- I Created grids for each item and added hyperlinks -->
 <template>
   <div>
     <main>
       <router-view />
     </main>
+    <!-- Footer -->
+    <footer class="page-footer font-small px-10 red pt-4 bg-light">
+        <!-- Footer Text -->
+        <div class="container-fluid text-center text-md-left">
+          <!-- Grid row -->
+          <div class="row">
+            <!-- Grid column -->
```

Comment:
Is the rest of the code commented with this level of detail?  A "Grid row" comment on a `div` with a class of `"row"` seems redundant to me.  If that's the convention throughout the project then it should be followed, but if not, I don't think all these comments are necessary.